src/Features/ERDDAP/Platform/Observations/CurrentConditions/data_card.tsx

```diff
@@ -118,26 +113,26 @@ export const DataCardDisplay: React.FunctionComponent<DataCardDisplayProps> = ({
 
   const data = readings.map((r) => ({
     ...r,
-    reading: round(data_converter.convertToNumber(r.reading, unit_system), 2),
+    reading: round(data_converter.convertToNumber(r.reading, unitSystem), 2),
   }))
 
   return (
     <Col {...cardProps}>
       <Link to={url}>
         <Card>
           <CardHeader>
-            {timeSeries.data_type.long_name} - {round(data_converter.convertToNumber(latest.reading, unit_system), 1)}{" "}
-            {data_converter.displayName(unit_system)} {convertUnit(timeSeries.data_type.units, latest.reading)}
+            {timeSeries.data_type.long_name} - {round(data_converter.convertToNumber(latest.reading, unitSystem), 1)}{" "}
+            {data_converter.displayName(unitSystem)} {convertUnit(timeSeries.data_type.units, latest.reading)}
           </CardHeader>
 
           <CardBody style={{ padding: ".2rem" }}>
             <SmallTimeSeriesChart
               name={timeSeries.data_type.standard_name}
               timeSeries={data}
-              unit={data_converter.displayName(unit_system)}
+              unit={data_converter.displayName(unitSystem)}
```

Comment:
Codacy found an issue: [Identifier 'data_converter' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6227039)